main.py

```diff
@@ -2,9 +2,49 @@
 
 from db.models import Race, Skill, Player, Guild
 
+import json
+
+
+def load_players(file_path: str) -> dict:
+    with open(file_path, "r") as file:
+        data = json.load(file)
+    return data
+
 
 def main() -> None:
-    pass
+    path_to_file = "players.json"
+    data_players = load_players(path_to_file)
+
+    for key, value in data_players.items():
```

Comment:
Better to use more descreptive names of variable, check what it is key and value in yout json file and change it)